File: library/e512io.cs
using System;
using System.IO;
using System.Linq;
using System.Numerics;
using System.Collections;
using System.Collections.Generic;

public class E512IO {
    public char separator = ' ';
    private string[] reads;
    private int index;
    public E512IO (bool autoflush = true) {
        StreamWriter sw = new StreamWriter(Console.OpenStandardOutput());
        sw.AutoFlush = autoflush;
        Console.SetOut(sw);
        this.reads = new string[0];
        this.index = 0;
    }
    ~E512IO () { Console.Out.Flush(); }
    private string NextValue () {
        this.index += 1;
        if (this.index > this.reads.Length - 1) {
            this.reads = Console.ReadLine().Split(separator);
            this.index = 0;
        }
        return this.reads[this.index];
    }
    private string[] NextLine () {
        this.reads = Console.ReadLine().Split(separator);
        this.index = this.reads.Length;
        return this.reads;
    }
    public int[] NextIntArray () { return this.NextLine().Select(x => int.Parse(x)).ToArray(); }
    public long[] NextLongArray () { return this.NextLine().Select(x => long.Parse(x)).ToArray(); }
    public double[] NextDoubleArray () { return this.NextLine().Select(x => double.Parse(x)).ToArray(); }
    public string[] NextStringArray () { return this.NextLine(); }
    public List<int> NextIntList () { return this.NextLine().Select(x => int.Parse(x)).ToList(); }
    public List<long> NextLongList () { return this.NextLine().Select(x => long.Parse(x)).ToList(); }
    public List<double> NextDoubleList () { return this.NextLine().Select(x => double.Parse(x)).ToList(); }
    public List<string> NextStringList () { return this.NextLine().ToList(); }
    public int NextInt () { return int.Parse(this.NextValue()); }
    public long NextLong () { return long.Parse(this.NextValue()); }
    public double NextDouble () { return double.Parse(this.NextValue()); }
    public string NextString () { return this.NextValue(); }
}

public class Program {
    static void Main (string[] args) {
        // E512IO io = new E512IO(false);// AutoFulsh
        E512IO io = new E512IO();
        // io.separator = ',';
        
        var n = io.NextInt();
        Console.WriteLine(n);
    }
}

File: library/e512io.vb
Imports System
Imports System.IO
Imports System.Collections.Generic
 
public class E512IO
    public separator as char = " "c
    private reads() as String
    private index as Int32
    public sub new (Optional ByVal autoflush as boolean = true)
        dim sw = new StreamWriter(Console.OpenStandardOutput())
        sw.AutoFlush = autoflush
        Console.SetOut(sw)
        me.reads = new String(){}
        me.index = 0
    end sub
    protected overrides sub Finalize ()
        Console.Out.Flush()
    end sub
    private function NextValue () as String
        me.index += 1
        if me.index > me.reads.Length - 1 then
            dim t() as String = Console.ReadLine().Split(separator)
            me.reads = t
            me.index = 0
        end if
        return me.reads(me.index)
    end function
    public function NextIntArray () as Int32()
        dim t() as String = Console.ReadLine().Split(separator) : me.index = t.Length
        dim r(t.Length-1) as Int32
        for i as Int32 = 0 to t.Length - 1 : r(i) = Int32.Parse(t(i)) : next i
        return r
    end function
    public function NextLongArray () as Int64()
        dim t() as String = Console.ReadLine().Split(separator) : me.index = t.Length
        dim r(t.Length-1) as Int64
        for i as Int32 = 0 to t.Length - 1 : r(i) = Int64.Parse(t(i)) : next i
        return r
    end function
    public function NextDoubleArray () as Double()
        dim t() as String = Console.ReadLine().Split(separator) : me.index = t.Length
        dim r(t.Length-1) as Double
        for i as Int32 = 0 to t.Length - 1 : r(i) = Double.Parse(t(i)) : next i
        return r
    end function
    public function NextStringArray () as String()
        dim t() as String = Console.ReadLine().Split(separator) : me.index = t.Length
        return t
    end function
    public function NextIntList () as List(of Int32)
        dim t() as String = Console.ReadLine().Split(separator) : me.index = t.Length
        dim r as new List(of Int32)
        for i as Int32 = 0 to t.Length - 1 : r.Add(Int32.Parse(t(i))) : next i
        return r
    end function
    public function NextLongList () as List(of Int64)
        dim t() as String = Console.ReadLine().Split(separator) : me.index = t.Length
        dim r as new List(of Int64)
        for i as Int32 = 0 to t.Length - 1 : r.Add(Int64.Parse(t(i))) : next i
        return r
    end function
    public function NextDoubleList () as List(of Double)
        dim t() as String = Console.ReadLine().Split(separator) : me.index = t.Length
        dim r as new List(of Double)
        for i as Int32 = 0 to t.Length - 1 : r.Add(Double.Parse(t(i))) : next i
        return r
    end function
    public function NextStringList () as List(of String)
        dim t() as String = Console.ReadLine().Split(separator) : me.index = t.Length
        dim r as new List(of String)
        for i as Int32 = 0 to t.Length - 1 : r.Add(t(i)) : next i
        return r
    end function
    public function NextInt () as Int32
        return Int32.Parse(me.NextValue())
    end function
    public function NextLong () as Int64
        return Int64.Parse(me.NextValue())
    end function
    public function NextDouble () as Double
        return Double.Parse(me.NextValue())
    end function
    public function NextString () as String
        return me.NextValue()
    end function
end class

class Program
    shared sub main ()
        ' dim io as new E512IO(false)' AutoFulsh false
        dim io as new E512IO()
        ' io.separator = ","
        
        dim n = io.NextInt()
        Console.WriteLine(n)
    end sub
end class

